app/charts/map/wms-utils.ts

```diff
@@ -109,7 +109,7 @@ export const getWMSTile = ({
   beforeId,
 }: {
   wmsLayers?: ParsedWMSLayer[];
-  customLayer?: WMSCustomLayer;
+  customLayer?: WMSCustomLayer | ParsedWMSLayer;
```

Comment:
That's more of a question, but maybe it should be renamed to `layer`? Not sure what's the difference between `CustomLayer` and `ParsedLayer` at this point, but maybe it's clear in the code 👍

Reply:
WMSCustomLayer is inside the config, only minimal information (endpoint, id etc..). ParsedWMSLayer is what's parsed from the XML, contains more information like attribution, title etc..